### PR TITLE
Add new WsFederation package

### PR DESCRIPTION
**Pull request by Tratcher**
See https://github.com/aspnet/Security/pull/1639

### CODE REVIEW

File: build/artifacts.props [hunk 1]
```diff
@@ -46,6 +46,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Authentication.OAuth" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Authentication.Twitter" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
+    <PackageArtifact Include="Microsoft.AspNetCore.Authentication.WsFederation" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
```
Comment (Eilon) [inline build/artifacts.props]: @DamianEdwards - just checking: this brings in some additional Microsoft-built dependencies, such as Microsoft.IdentityModel.Protocols.WsFederation and others. Is that OK for the .App package?

Comment (Tratcher) [reply build/artifacts.props]: They're largely the same dependencies as OpenIdConnect.

**Comment by natemcmaster** [comment]
Actually, this will most likely break Universe. You also need to update the submodule that contains this package.

File: build/artifacts.props (hunk 1 above)
Comment (Eilon) [reply build/artifacts.props]: Ah ok then I'm not concerned. Thanks!

**Pull request merged**